README.md

```diff
@@ -415,7 +415,34 @@ Logger.getInstance().setLogLevel(Logger.LogLevel.Verbose);
   adb logcat > "C:\logmsg\logfile.txt"
  ```
  More examples about adb cmds: https://developer.android.com/tools/debugging/debugging-log.html#startingLogcat
- 
+
+#### Telemetry
+
+ADAL provides a built-in callback mechanism to supply consuming applications with event data generated during requests. The event data is sanitized of Personally Identifiable Information (PII) and Organizationally Identifiable Information (OII) and is designed to give consumers of the library insight into the performance, reliability, and usage of ADAL.
+
+To capture telemetry data, add the following to your application subclass:
+```java
+// Get a reference to the Telemetry singleton
+private static final Telemetry sTelemetry = Telemetry.getInstance();
+
+// Flag to turn event aggregation on/off
+private static final boolean sTelemetryAggregationIsRequired = false;
```

Comment:
>sTelemetryAggregationIsRequired [](start = 29, length = 31)

can you please add more information explaining aggregation and what true vs false does with examples?